sentry-spring/src/main/java/io/sentry/spring/tracing/SentrySpanClientHttpRequestInterceptor.java

```diff
@@ -41,10 +41,9 @@ public SentrySpanClientHttpRequestInterceptor(final @NotNull IHub hub) {
       return execution.execute(request, body);
     }
 
-    final ISpan span = activeSpan.startChild();
+    final ISpan span = activeSpan.startChild("http");
```

Comment:
not really related to this PR but: https://github.com/getsentry/develop/issues/242
shouldn't it be `http.server` then?

Reply:
makes sense, should @Tyrrrz align outgoing requests?

Reply:
FWIW Python uses `"http"` https://github.com/getsentry/sentry-python/blob/ca828c9486f4a0030a744849a4e8e10c5a1c7aeb/sentry_sdk/integrations/stdlib.py#L81

Reply:
`http.client` and `http.server` make sense to me. Is this something we can change Python into alignment or we just deviate here? /cc @untitaker and @brustolin for visibility

Reply:
I'd also prefer `http.client` as `http` is just ambiguous.
I will create an issue for that so we can move ahead with this PR

Reply:
https://github.com/getsentry/develop/issues/247